entries/c/cloud.baidu.com.json

```
@@ -0,0 +1,17 @@
+{
+    "Baidu AI cloud": {
+      "domain": "cloud.baidu.com",
+      "tfa": [
+        "totp",
+        "custom-software"
+      ],
+      "custom-software": [
+        "Baidu AI Cloud APP"
+      ],
+      "documentation": "https://intl.cloud.baidu.com/doc/IAM/s/2jwvyc50i-en",
+      "categories": [
+        "cloud"
+      ]
```

Even though the service isn't restricted to mainland China, the login page is mostly in Chinese regardless of the selected language, and there are no English terms of service or privacy policies. Furthermore, there's no mention of GDPR or U.S. equivalents, which would rule out those markets completely. 
I therefore suggest adding a regions element labeling China as the main audience:
```suggestion
      ],
      "regions": [
        "cn"
      ]
```

---

Yes, but I found another [domain](https://intl.cloud.baidu.com/) offer English resource to users. How should I deal with it?

---

Yeah I found that page as well but my the lack of required legal documents and the login page being 50% Chinese still stands.
Therefore, please include the regions element as suggested.

---

Thanks for your suggestion!